.templates/vscode/.fttemplates/form/types.ts

```diff
@@ -0,0 +1 @@
+export type <FTName | pascalcase>Values = {};
```

Comment:
https://kaluga-astral.github.io/style-guide/docs/rules/arch/modules/validations#%D0%B4%D0%BB%D1%8F-uselogic-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D0%B5%D1%82%D1%81%D1%8F-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-%D1%84%D0%B0%D0%B9%D0%BB-types-%D0%B4%D0%BB%D1%8F-%D0%B2%D0%B0%D0%BB%D0%B8%D0%B4%D0%B8%D1%80%D1%83%D0%B5%D0%BC%D1%8B%D1%85-%D0%B4%D0%B0%D0%BD%D0%BD%D1%8B%D1%85